demisto_sdk/commands/common/errors.py

```diff
@@ -1249,6 +1252,35 @@ def incident_type_non_existent_playbook_id(incident_type, playbook):
                f"2 - The id_set.json file is up to date. Delete the file by running: rm -rf Tests/id_set.json and" \
                f" rerun the command."
 
+    @staticmethod
+    @error_code_decorator
+    def incident_field_non_existent_script_id(incident_field, scripts):
+        return f"In incident field {incident_field} the following scripts were not found in the id_set.json file:" \
+               f" {scripts} \n " \
+               f"Please make sure:\n" \
+               f"1 - The right script name is set and the spelling is correct.\n" \
+               f"2 - The id_set.json file is up to date. Delete the file by running: rm -rf Tests/id_set.json and" \
+               f" rerun the command with the --create-id-set option."
```

Comment:
Let's put this in a single method and use the `suggested_fix` feature of `handle_error` in order to avoid needing to update all of these errors in case a new solution arises.